libc/hdr/unistd_overlay.h

```diff
@@ -0,0 +1,69 @@
+//===-- Including unistd.h in overlay mode ---------------------------------===//
```

Comment:
nit: this line is too long

Reply:
How should we fix it since clang-format probably is complaining about other header files with header lines that are too long? Just letting clang-format do it's thing makes it look awkward.

Reply:
Fixed it.